app/models/local_authority.rb

```diff
@@ -15,4 +15,31 @@ class LocalAuthority < ResponsibleBody
   }
 
   validates :organisation_type, presence: true
+
+  def la_funded_place
+    schools.la_funded_place_establishment_type&.first
+  end
+
+  def create_la_funded_places!(urn:, device_allocation: 0, router_allocation: 0, extra_args: {})
+    return unless la_funded_place.nil?
```

Comment:
I'd probably make that `return if la_funded_place.present?`

Reply:
fixed